tests/_test_core_strings.py

```diff
@@ -188,3 +193,168 @@
 
 }); // END REQUIRE.JS block
 </script>"""
+
+STANDALONE_HTML_STRING = """
```

Comment:
K, was wondering if it will be an issue with pep8 but it seems like we are fine.